### PR TITLE
create task_15

### PR DESCRIPTION
**Pull request by TanyaLevchenya**

### CODE REVIEW

File: task_15/task_2.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+let temp;
+
+temp = a;
+a = b;
+b = temp;
+
+console.log(a);
+console.log(b);
+
+let a = 20;
```
Comment (Killerkip10) [inline task_15/task_2.js]: You initialize the same variable in one file!
Error will be thrown

File: task_15/task_4.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+let firstNum = +prompt();
+let secondOperations = prompt();
+let thirdNum = +prompt();
+
+if (secondOperations == '-'){
+        alert (firstNum - thirdNum);
```
Comment (Killerkip10) [inline task_15/task_4.js]: alert(firstNum - thirdNum);

Without space!

File: task_15/task_5.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+let query = prompt('Your name?');
+let name = 'Hello ';
```
Comment (Killerkip10) [inline task_15/task_5.js]: This variable is redundant

File: task_15/task_7.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+'use strict'
+let result;
+
+let numbers = prompt();
+let sum = 0;
+console.log(numbers);
+
+let i = 0;
+while (i < numbers.length) {
+    if(i == 0) {
```
Comment (Killerkip10) [inline task_15/task_7.js]: If you set initial value `let sum = 1` you will remove this `if`

**Review by Killerkip10 — COMMENTED** [review]
## Task 14
### Total 100 points 

Please resolve all conversations 
_**before 25.09.2020 23:59 or -20 points**_

File: task_15/task_2.js (hunk 1 above)
Comment (TanyaLevchenya) [reply task_15/task_2.js]: correct

File: task_15/task_7.js (hunk 4 above)
Comment (TanyaLevchenya) [reply task_15/task_7.js]: correct

File: task_15/task_5.js (hunk 3 above)
Comment (TanyaLevchenya) [reply task_15/task_5.js]: correct

File: task_15/task_4.js (hunk 2 above)
Comment (TanyaLevchenya) [reply task_15/task_4.js]: correct

File: task_15/task_5.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+let query = prompt('Your name?');
+
+if (query.trim() == '') {
+    alert('Invalid Name');
+}
+else {
+    alert(' Hello ' + name + query);
```
Comment (Killerkip10) [inline task_15/task_5.js:9]: What is name?)

**Review by Killerkip10 — COMMENTED** [review]
## Task 15
### Total 100 points

**Pull request merged**